src/wdl/tasks/terra_tasks.wdl

```diff
@@ -247,4 +249,35 @@ task addReadsField {
     output {
         String reads_json = read_string("reads.json")
     }
+}
+
+task ValidateFileStatus {
+    input {
+        String program
+        String project
+        String sample_alias
+        String agg_project
+        String data_type
+        String gdc_token
+        Boolean previous_task
```

Comment:
What is the `previous_task` parameter used for?

Reply:
Its used in the main wdl, so it waits for the transfer step to finish

Reply:
Ohh I missed that, ok got it!